wg-batch/charter.md

```diff
@@ -0,0 +1,73 @@
+# WG Batch Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Discuss and enhance the support for Batch (eg. HPC, AI/ML, data analytics) workloads
+in core Kubernetes. We want to unify the way users deploy batch workloads to improve
+portability and to simplify supportability for Kubernetes providers.
+
+### In scope
+
+- Additions to the batch API group, currently including Job and CronJob resources
+  that benefit batch use cases such as HPC, AI/ML and data analytics.
+- Primitives for job-level queueing, not limited to the k8s Job resource.
+- Primitives to control and maximize utilization of resources in fixed-size clusters (on-prem) and elastic clusters (cloud).
+- Runtime and scheduling support for specialized hardware (GPUs, NUMA, RDMA, etc.)
+
+### Out of scope
+
+- Uses of the batch APIs as support for serving workloads (eg. backups,
+  upgrades, migrations).
+- Proposals that duplicate the functionality of core kubernetes components
+  (job-controller, kube-scheduler, cluster-autoscaler).
+- Job workflows or pipelines. Mature third party frameworks serve these
+  use cases with the current kubernetes primitives. But additional primitives
+  to support these frameworks could be in scope.
+
+## Stakeholders
+
+Stakeholders in this working group span multiple SIGs that own parts of the
+code in core kubernetes components and addons.
+
+- Apps
+- Autoscaling
+- Node
+- Scheduling
+
+## Deliverables
+
+The list of deliverables is TBD, but here are some high level features:
+
+- To SIG Apps:
+  - Updated Job API that fulfills the needs of a wider range of batch applications.
+  - A performant job controller that can scale to thousands of pods per minute.
```

Comment:
Cool! Serverless may benefit from this.

Reply:
This will probably also need to result in node-level performance enhancements too.

Reply:
I don't think node-level performance improvements are a exclusive need for batch. I think they can be looked at by SIG Node independently. WDYT?

Reply:
can this be resolved?

Reply:
@ahg-g i think so. @endocrimes let's tackle those separately as needed.